lte/gateway/c/core/oai/tasks/amf/amf_smf_send.cpp

```diff
@@ -362,13 +359,13 @@ static int pdu_session_resource_release_t3592_handler(
 
 /***************************************************************************
 **                                                                        **
-** Name:    amf_smf_send()                                                **
+** Name:    amf_smf_process_pdu_session_packet()                          **
 **                                                                        **
 ** Description: handler to send session request to SMF                    **
 **                                                                        **
 **                                                                        **
 ***************************************************************************/
-int amf_smf_send(
+int amf_smf_process_pdu_session_packet(
     amf_ue_ngap_id_t ue_id, ULNASTransportMsg* msg, int amf_cause) {
   int rc = 1;
```

Comment:
Please define a macro to indicate what 1 means (success/failure).

Reply:
Addressed review comment.